lighthouse-core/test/report/html/renderer/details-renderer-test.js

```diff
@@ -548,7 +548,7 @@ describe('DetailsRenderer', () => {
         // itemType is overriden by code object
         headings: [{key: 'content', itemType: 'url', text: 'Heading'}],
         items: [
-          {content: {type: 'code', value: 'code object'}},
+          {content: {type: 'code', value: 'https://codeobject.com'}},
```

Comment:
The fallback code in the details render will render a URL w/ an invalid url as a `code` type - so, this test could possible have a false positive (if we ever refactored and broke the fallback type mechanism).